setup.js

```diff
@@ -0,0 +1,6 @@
+import 'dotenv/config';
+import { client } from './src/utils/db.js';
+import './src/models/user.js';
+import './src/models/token.js';
+
+client.sync({ force: true });
```

Comment:
Using `client.sync({ force: true })` will drop all tables and recreate them. This is usually used in development environments. Ensure this is not used in production as it will lead to data loss.